Yoast/wordpress-seo
# Add content modifications to pluggable data modifications

## Summary

This PR can be summarized in the following changelog entry:

* n/a

## Relevant technical choices:

* Adds more modifications to the pluggable data modifications.

## Test instructions

This PR can be tested by following these steps:

* For the content modifications, test e.g. whether analyses work in AFC. For this, you need Advanced Custom Fields plugin and the Yoast AFC plugin.
* In the Advanced Custom Fields plugin, create a new field.
* Make a post and set a keyword. Make sure you have enough words (> 150) to trigger the keyword count assessment.
* Add your keyword in the ACF field. Make sure it's counted towards the overall keyword count.

## Quality assurance


Fixes https://github.com/Yoast/YoastSEO.js/issues/1710
